mta/mta.sw.yaml

```diff
@@ -42,7 +42,7 @@ states:
         actionDataFilter:
           toStateData: ".applications"
     stateDataFilter:
-      output: "{backstageUser: .backstageUser, backstageGroup: .backstageGroup, repositoryURL: .repositoryURL, application: .repositoryURL as $r| if .applications == null then null else .applications|map(select(.repository.url == $r))[0] end}"
+      output: "{repositoryURL: .repositoryURL, application: .repositoryURL as $r| if .applications == null then null else .applications|map(select(.repository.url == $r))[0] end}"
```

Comment:
why not adding the `.recipients`? as you did https://github.com/parodos-dev/serverless-workflows/pull/297/files#diff-dc5c3c5cf797d197de6c902e605c4d693eb7b895ed40073983a8c66df0e30ac6R81